cypress/integration/26-register_publication/543-creator_begins_wizard_registration_and_navigates_to_description_tab.feature

```diff
@@ -0,0 +1,28 @@
+  Feature: Creator begins Wizard registration and navigates to Description tab
+
+  Scenario: Creator begins Wizard registration and navigates to Description tab
+    Given Creator begins registering a Registration in the Wizard
+    When they navigate to the Description tab
+    Then they see the Description tab is selected
+    And they see fields:
+      | Title |

```

Comment:
ser ut som det mangler formattering her. Kanskje vi skulle vurdere å formatere koden på en precommit hook?

Reply:
Skal se på config i Visual om jeg kan fikse noe der

Reply:
hvis du bruker vscode fikk jeg også problemer med dette for noen uker siden. måtte gå på settings og endre default formatter, da den ikke skjønte at det skulle bytte fra prettier til den Gherkin-plugin